experimental/regression_suite/shark-test-suite-models/llama/test_llama.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers_tools import *
+import os
+from conftest import VmfbManager
+from pathlib import Path
+
+rocm_chip = os.getenv("ROCM_CHIP", default="gfx90a")
+vmfb_dir = os.getenv("TEST_OUTPUT_ARTIFACTS", default=Path.cwd())
+
+###############################################################################
+# Fixtures
+###############################################################################
+llama_real_weights = fetch_source_fixture(
+    "https://sharkblobs.blob.core.windows.net/halo-models/llm-dev/llama3_8b/8b_f16.irpa",
+    group="llama_fp16_8b",
+)
```

Comment:
These tests should use public files. If we can't share the real weights publicly for llama, can we use fake weights? Or can the workflow download from huggingface using HF_TOKEN?

Reply:
ah good catch, I'll migrate this to sharkpublic